core/script.runtime/src/kotlin/script/extensions/samWithReceiver.kt

```diff
@@ -16,6 +16,11 @@
 
 package kotlin.script.extensions
 
+// discuss
+
+// Is this an appropriate place to put this class?
```

Comment:
Yes, that would be a generic solution for such problems, I like it. But then we should implement it before dropping this annotation from here.